cmd/zoekt-webserver/main.go

```diff
@@ -474,59 +473,14 @@ func traceID(ctx context.Context) string {
 
 // traceIDFromSpan returns a trace ID, if any, found in the given span.
 func traceIDFromSpan(span opentracing.Span) string {
-	spanCtx, ok := span.Context().(jaeger.SpanContext)
-	if !ok {
-		return ""
+	if spanCtx, ok := span.Context().(jaeger.SpanContext); ok {
```

Comment:
nit: you can use a `switch span.Context().(type)` to more efficiently check the types of things.